FlowCrypt/Core/Core.swift

```diff
@@ -211,18 +213,22 @@ final class Core: KeyDecrypter, CoreComposeMessageType {
         try blockUntilReadyOrThrow()
         cb_last_value = nil
         jsEndpointListener!.call(withArguments: [endpoint, String(data: jsonData, encoding: .utf8)!, data.base64EncodedString(), cb_catcher!])
-        let b64response = cb_last_value![0] as! String
-        let rawResponse = Data(base64Encoded: b64response)!
-        let separatorIndex = rawResponse.firstIndex(of: 10)!
-        let resJsonData = Data(rawResponse[...(separatorIndex - 1)])
+        guard
+            let resJsonData = cb_last_value?.0.data(using: .utf8),
+            let rawResponse = cb_last_value?.1
+        else {
+            throw CoreError.format("JavaScript callback response not available")
+        }
+
         let error = try? resJsonData.decodeJson(as: CoreRes.Error.self)
```

Comment:
Now we have the same error handler for all methods in JavaScript.

Reply:
Yes but test are failing - please have a look further.

Reply:
added - try now

Reply:
I forgot to add `FlowCrypt/Resources/flowcrypt-ios-prod.js.txt` because I was thinking you generate it from sources

Reply:
Sorry about that - we don't have an automated process for that yet, for now it has to be manually updated with `npm run-script deploy` and checked into the repo.

Once we auto-generate it from sources during build, we'll definitely gitignore it, to avoid confusion.